pkg/syncer/metadatasyncer.go

```diff
@@ -1567,8 +1588,37 @@ func csiPVUpdated(ctx context.Context, newPv *v1.PersistentVolume, oldPv *v1.Per
 		}
 		volumeOperationsLock.Lock()
 		defer volumeOperationsLock.Unlock()
+
+		// If it is a multi VC deployment, figure out FCD's location based on PV's nodeAffinity rules.
+		if isMultiVCenterFssEnabled && len(metadataSyncer.configInfo.Cfg.VirtualCenter) > 1 {
+			vcHost, cnsVolumeMgr, err = getVcHostAndVolumeManagerFromTopologySegments(ctx, newPv, metadataSyncer)
+			if err != nil {
+				log.Errorf("PVDeleted: Failed to get VC host and volume manager for single VC setup. "+
```

Comment:
replace PVDeleted with csiPVUpdated in log message. also check other places wherever applicable